plugins/modules/oneview_label_facts.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2021) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_label_facts
+short_description: Retrieve the facts about one or more of the OneView Labels.
+description:
+    - Retrieve the facts about one or more of the Labels from OneView.
+version_added: "2.5.0"
+requirements:
+    - hpeOneView >= 5.4.0
```

Comment:
same versioning is missing

Reply:
updated

Reply:
labels resource is refactored in python 6.1.0. So please mention the same

Reply:
okay, Thanks, have updated.